app/src/main/res/values/edit_name_list_strings.xml

```diff
@@ -39,7 +39,8 @@
     <string name="multiple_deletions_title">How many copies of this name do you want to delete?</string>
     <string name="names_deleted">Name(s) deleted.</string>
 
-    <string name="multiple_renames_title">How many copies of this name do you want to change?</string>
+    <string name="multiple_renames_title">How many copies of \'%s\' do you want to change? You currently have %s copies of this name.</string>
```

Comment:
Let's split this up, so it doesn't flood people's IDEs. Also, we should use %d for numbers. Lastly, we should use double quotes, not single quotes. Double quotes are kind of wonky in XML though, so you might need to just use %s and then add the quotes in Java.
```suggestion
    <string name="multiple_renames_title">
        How many copies of \'%s\' do you want to change?
        You currently have %d copies of this name.
    </string>
```